src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -54,7 +54,9 @@ public static SqlConnection Create(IPersistentStorageFaultInjector faultInjector
             // Enable shared cache so that multiple connections inside of same process share cache
             // see https://sqlite.org/threadsafe.html for more detail
             var flags = OpenFlags.SQLITE_OPEN_CREATE | OpenFlags.SQLITE_OPEN_READWRITE | OpenFlags.SQLITE_OPEN_NOMUTEX | OpenFlags.SQLITE_OPEN_SHAREDCACHE;
+#pragma warning disable CA2000 // Dispose objects before losing scope - handle is wrapped with the returned SqlConnection
```

Comment:
❓ How/why is this an ownership transfer? The resulting object does not appear to be disposable.